src/core/atomic.d

```diff
@@ -1142,6 +1142,45 @@ version (CoreUnittest)
             assert(atomicOp!"+="(i64, 8) == 16);
         }
     }
+    
+    @betterC pure nothrow @nogc unittest
+    {
+        byte[] byteArray = [1, 3, 5, 7, 9, 11, 13, 15, 17, 19];
+        ulong[] ulongArray = [2, 4, 6, 8, 10, 12, 14, 16, 19, 20];
+
+        {
+            auto array = byteArray;
+            byte* ptr = &array[0];
+            byte* prevPtr = atomicFetchAdd(ptr, 3); 
+            assert(prevPtr == &array[0]);
+            assert(*prevPtr == 1);
+            assert(*ptr == 7);
```

Comment:
The code style check fails because you have mixed tabs and spaces indentation. Please convert all tabs to 4 space and we should be good.

Reply:
I cannot locate any tabs in the file.

Reply:
That's strange... I was checking this PR through my phone and it seemed to me that the line above had tabs, but now when checking on my desktop it doesn't seem to be the case. The error message on [CircleCI](https://app.circleci.com/pipelines/github/dlang/druntime/3410/workflows/5123f4ac-4197-423f-94fa-aa7e8b65cd84/jobs/7388) is:

```
grep -n -U -P "( $|\r|\t)" $files ; test "$?" -ne 0
src/core/atomic.d:1145:    
src/core/atomic.d:1154:            byte* prevPtr = atomicFetchAdd(ptr, 3); 
src/core/atomic.d:1162:            shared(byte)* prevPtr = atomicFetchAdd(ptr, 3); 
src/core/atomic.d:1170:            ulong* prevPtr = atomicFetchAdd(ptr, 3); 
src/core/atomic.d:1178:            shared(ulong)* prevPtr = atomicFetchAdd(ptr, 3); 
src/core/atomic.d:1217:            byte* prevPtr = atomicFetchSub(ptr, 4); 
src/core/atomic.d:1225:            shared(byte)* prevPtr = atomicFetchSub(ptr, 4); 
src/core/atomic.d:1233:            ulong* prevPtr = atomicFetchSub(ptr, 4); 
src/core/atomic.d:1241:            shared(ulong)* prevPtr = atomicFetchSub(ptr, 4); 
posix.mak:363: recipe for target 'checkwhitespace' failed
```

So the issue is that you have trailing whitespace:

![image](https://user-images.githubusercontent.com/3475150/105690144-4e1d8f00-5f04-11eb-973c-8a5cbebe5fc3.png)

By the way, if you install [EditorConfig](https://editorconfig.org/) on your editor of choice it should fix such things automatically, based on the `.editorconfig` file in our repo: https://github.com/dlang/druntime/blob/master/.editorconfig

Reply:
Fixed. Though it still fails somewhere.
Is there a way to see the CircleCI output without giving them access to by github account?

Reply:
No, as far as I know